pymc_experimental/inference/quadratic.py

```diff
@@ -0,0 +1,140 @@
+from collections.abc import Sequence
+from typing import Optional
+
+import arviz as az
+import numpy as np
+import pymc as pm
+import xarray as xr
+from arviz import dict_to_dataset
+from pymc.backends.arviz import (
+    coords_and_dims_for_inferencedata,
+    find_constants,
+    find_observations,
+)
+from pymc.util import RandomSeed
+from pytensor import Variable
+
+
+def quadratic(
+    vars: Sequence[Variable],
+    draws=1_000,
+    model=None,
+    random_seed: Optional[RandomSeed] = None,
+):
+    """
+    Create a quadratic approximation for a posterior distribution.
```

Comment:
Mention laplace somewhere early in the docstrings. Some folks have expressed a preference for calling it laplace actually (in which case quadratic should be mentioned early in the docstrings). I don't have a preference myself.

Reply:
I would prefer Laplace. numpyro also use Laplace https://docs.pyro.ai/en/1.4.0/infer.autoguide.html#autolaplaceapproximation